app/pages/sections/catalogue-solutions/routes.js

```diff
@@ -93,8 +95,19 @@ export const catalogueSolutionsRoutes = (authProvider, addContext, sessionManage
 
   router.get('/select-solution/select-price/select-recipient', authProvider.authorise({ claim: 'ordering' }), withCatch(authProvider, async (req, res) => {
     const { orderId } = req.params;
+    const accessToken = extractAccessToken({ req, tokenType: 'access' });
+    const solutionId = sessionManager.getFromSession({ req, key: 'selectedSolution' });
+
+    const solutionData = await getSolution({ solutionId });
 
-    const context = await getSolutionRecipientPageContext({ orderId, solutionName: 'Solution One' });
+    const recipients = await getRecipients({ orderId, accessToken });
+    sessionManager.saveToSession({ req, key: 'recipientsFound', value: recipients });
```

Comment:
`recipientsData` instead of recipientsFound?

Reply:
I've decided to call it recipientsFound to keep it consistent with the other similar keys we have for similar pages like solutionsFound, suppliersFound